opm/autodiff/AquiferCarterTracy.hpp

```diff
@@ -300,7 +300,18 @@ namespace Opm
                             // Check now if the face is outside of the reservoir, or if it adjoins an inactive cell
                             // Do not make the connection if the product of the two cellIdx > 0. This is because the
                             // face is within the reservoir/not connected to boundary. (We still have yet to check for inactive cell adjoining)
-                            faceArea = (faceCells(faceIdx,0)*faceCells(faceIdx,1) > 0)? 0. : Opm::UgGridHelpers::faceArea(ugrid, faceIdx);
+                            auto cellNeighbour0 = faceCells(faceIdx,0);
```

Comment:
This part of the code can be put into a function to calculate face area, so the following code can become

```
faceArea_connected_.at(idx) = faceArea(.....);
```